.github/workflows/test.yaml

```diff
@@ -283,7 +283,7 @@ jobs:
           E2E_NEW_USER_STORAGE: ${{ secrets.E2E_NEW_USER_STORAGE}}
           HAPPO_API_KEY: ${{ secrets.HAPPO_API_KEY }}
           HAPPO_API_SECRET: ${{ secrets.HAPPO_API_SECRET }}
-          HAPPO_NONCE: ${{ github.event.pull_request.head.sha }}
+          HAPPO_NONCE: ${{ github.sha }}
```

Comment:
In retrospect it makes sense this wouldn't work after merging into main since there is no pull request sha. So I'm using `github.sha`:

<img width="722" alt="Screenshot 2024-03-20 at 2 26 37 PM" src="https://github.com/bcgov/cas-registration/assets/14259474/e7940b53-95b0-4a7c-b930-bc1a71899ce6">

Reply:
@marcellmueller and now you are an expert in yaml as well- nice!